### PR TITLE
Update allow-direct-connect-with-all-organizations.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note stating that the guest access in groups should be enabled for this to work

Per issue #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9361

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/solutions/allow-direct-connect-with-all-organizations.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ If your organization doesn't have a requirement to restrict collaboration with o
 > [!NOTE]
 > Changes to cross-tenant access settings may take two hours to take effect.
 
+> [!NOTE]
+> The groups should also be enabled for guest access. For more informaton, read [Manage groups guest access](https://docs.microsoft.com/en-us/microsoft-365/admin/create-groups/manage-guest-access-in-groups)
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/allow-direct-connect-with-all-organizations.md]:
```suggestion
> The groups should also be enabled for guest access. For more information, read [Manage guest access in Microsoft 365 Groups](/microsoft-365/admin/create-groups/manage-guest-access-in-groups).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Pull request merged**